.github/workflows/publish-sources.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish
+
+on:
+  push:
+    branches: # Snapshots
+      - 2.9.x
+
+jobs:
+  publish-sources:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Package sources
+        run: |
+          ./sources.sh
+      - name: Delete Release
+        run: gh release delete 2.9.x-nightly --cleanup-tag || true
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish nightly sources
+        uses: softprops/action-gh-release@v1
+        with:
+          prerelease: true
+          name: 2.9.x-nightly
```

Comment:
@mkurz WDYT about this release/tag name? It will be a part of download link path, as you understand 😄

Reply:
Actually I think just using `2.9.x` (and `3.0.x` in the other PR) is better. Also the tag here should not be called `*-nightly` but just `2.9.x` (or `v2.9.x` to better distinguish it from the `2.9.x` branch? actually technically it would be possible to name a branch and a tag the same name).

The reason why I would avoid `nightly` here is because 2.9.x and 3.0.x are actually the "stable" branches of the samples.

So in the main branch url is
```
https://github.com/playframework/play-samples/releases/download/nightly/play-java-chatroom-gradle-example.zip
```

and I think here, for the 2.9.x branch, it should be
```
https://github.com/playframework/play-samples/releases/download/[v]2.9.x/play-java-chatroom-gradle-example.zip
```

Because clearly you download a (stable!) sample for a Play 2.9.x release. And since we update the samples every now and then (dependency upgrades etc.) I think using 2.9.x makes total sense, so users always download the currently up-to-date sample for the stable 2.9.x release.

Reply:
(also please see my other comment: https://github.com/playframework/play-samples/pull/569#issuecomment-1977418671)

Reply:
> or `v2.9.x` to better distinguish it from the `2.9.x` branch? actually technically it would be possible to name a branch and a tag the same name

Yes, technically we can use the same name for tag but I wanted to avoid this potentially confusing option.
OMG, this option with `v` prefix was a very straightforward, how I didn't catch them 🤦‍♂️

Reply:
Changed to `v2.9.x`. Thanks, Matthias! 👍